demos/ota/ota_demo_core_mqtt/ota_demo_core_mqtt.c

```diff
@@ -136,7 +136,7 @@
 /**
  * @brief Period for demo loop sleep in milliseconds.
  */
-#define OTA_EXAMPLE_LOOP_SLEEP_PERIOD_MS    ( 5000U )
+#define OTA_EXAMPLE_LOOP_SLEEP_PERIOD_MS    ( 5U )
```

Comment:
5 milliseconds is very fast.  Why are we reducing this 1000x?

Reply:
We actually intended to use 5 ms. This macro was earlier used with `usleep()` that takes microseconds and this config was not updated when we shifted to `clock_posix` : `Clock_SleepMs`.